### PR TITLE
Add Utc test to ensure creating only one instance

### PR DESCRIPTION
**Pull request by tarekgh**

### CODE REVIEW

**Comment by tarekgh** [comment]
This PR depends on https://github.com/dotnet/coreclr/pull/24614

File: src/System.Runtime/tests/System/TimeZoneInfoTests.cs [hunk 1]
```diff
@@ -2243,6 +2244,14 @@ public static void TimeZoneInfo_DisplayNameStartsWithOffset()
             }
         }
 
+        [Fact]
+        public static void EnsureUtcObjectSingleton()
```
Comment (bartonjs) [inline src/System.Runtime/tests/System/TimeZoneInfoTests.cs]: Won't it need a [SkipOnTargetFramework] to exclude NetFx?

Comment (tarekgh) [reply src/System.Runtime/tests/System/TimeZoneInfoTests.cs]: good point. I'll add the attribute.

**Comment by tarekgh** [comment]
The CI failure is known issue tracked by https://github.com/dotnet/corefx/issues/31608. Merging the changes.

**Pull request merged**